build-system/runner/src/org/ampproject/AmpCodingConvention.java

```diff
@@ -45,9 +46,8 @@ public AmpCodingConvention(CodingConvention convention) {
     return ImmutableList.of(
         new AssertionFunctionSpec("user.assert", JSType.TRUTHY),
         new AssertionFunctionSpec("dev.assert", JSType.TRUTHY),
-        new AssertionFunctionSpec("module$src$log.user.assert", JSType.TRUTHY),
-        new AssertionFunctionSpec("module$src$log.dev.assert", JSType.TRUTHY),
-        new AssertionFunctionSpec("Log$$module$src$log.prototype.assert", JSType.TRUTHY)
+        new AssertionFunctionSpec("Log$$module$src$log.prototype.assert", JSType.TRUTHY),
```

Comment:
ah nvm, looks like that doesn't work. i was hoping this `AssertFunctionByTypeName` solves the inference issue